wordpress-mobile/WordPress-iOS
# Add reminders schedule syncing for Jetpack

## Description

Sync blogging reminders with Jetpack by:

- WP: Copying reminders schedule from local file to shared directory
- JP: Copying shared reminders schedule to local file

## Testing

To test:

- Make sure `sharedLogin` is set to `false`
- Launch WordPress
- Login if necessary
- Navigate to Menu > Site Settings > Blogging Reminders
- Create a reminder
- Switch to another site
- Create a reminder with the above steps
- Change `sharedLogin` to `true`
- Relaunch WordPress (so the file can copy)
- Launch Jetpack
- Verify both sites have the same schedule as WordPress

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
